src/font/lv_font.c

```diff
@@ -71,12 +71,23 @@ bool lv_font_get_glyph_dsc(const lv_font_t * font_p, lv_font_glyph_dsc_t * dsc_o
     bool found = false;
     while(f) {
         found = f->get_glyph_dsc(f, dsc_out, letter, letter_next);
+        dsc_out->resolved_font = f;
         if(found && !dsc_out->is_placeholder) {
-            dsc_out->resolved_font = f;
             break;
         }
         f = f->fallback;
     }
+
+    if(found == false) {
+        dsc_out->adv_w = font_p->line_height / 2;
```

Comment:
Add `dsc_out->resolved_font = NULL` to not point to font which doesn't contain the glyph.